libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/listener/OracleModeParserListener.java

```diff
@@ -418,6 +418,26 @@ public void enterRollback_statement(PlSqlParser.Rollback_statementContext ctx) {
         doRecord(SqlType.ROLLBACK, DBObjectType.OTHERS, null);
     }
 
+    @Override
+    public void enterComment_on_column(PlSqlParser.Comment_on_columnContext ctx) {
+        doRecord(SqlType.COMMENT, DBObjectType.COLUMN, null);
+    }
+
+    @Override
+    public void enterComment_on_table(PlSqlParser.Comment_on_tableContext ctx) {
+        doRecord(SqlType.COMMENT, DBObjectType.TABLE, null);
+    }
+
+    @Override
+    public void enterComment_on_materialized(PlSqlParser.Comment_on_materializedContext ctx) {
+        doRecord(SqlType.COMMENT, DBObjectType.OTHERS, null);
+    }
+
+    @Override
+    public void enterCall_spec(PlSqlParser.Call_specContext ctx) {
```

Comment:
this ast node is wrong, I think what you wants to handle is `procedure_call`?

Reply:
removed enterCall_spec